index.html

```diff
@@ -2342,6 +2333,50 @@
           </li>
         </ol>
       </section>
+      <section>
+        <h2>
+          Filter payment options algorithm
+        </h2>
+        <p>
+          The <dfn data-lt="filter payment options">filter payment options
+          algorithm</dfn> runs when user agent presents the user with the
+          collection of payment options available for selection. It MUST run the
+          following steps:
+        </p>
+        <ol>
+          <li>Let <var>matchingOptions</var> be an empty list.</li>
+          <li>Let <var>request</var> be the <a>PaymentRequest</a> object that
+          the user is interacting with.
+          <li>For each <var>requestMethod</var> in
+            <var>request</var>.<a>[[\serializedMethodData]]</a>:
+            <ol type="I">
+              <li>Let <var>filters</var> be <var>requestMethod.data</var>
```

Comment:
This step doesn't make sense to me. requestMethod is a tuple containing supportedMethods, a `sequence<DOMString>`, and `data`, a string of serialized JSON. So `requestMethod.data` does not have any members.